weather_forecast_retrieval/hrrr_preprocessor.py

```diff
@@ -63,6 +74,29 @@ def __init__(self, hrrr_dir, start_date, end_date, output_dir,
         self._logger.info('Forecast hour: {}'.format(self.forecast_hr))
         self._logger.info('Number of cpu argument: {}'.format(self.ncpu))
 
+    @property
+    def variable_match(self):
+        return '|'.join(self.VARIABLES)
+
+    def check_for_good_file(self, file_name):
+
+        check_action = 'wgrib2 {}'.format(file_name)
```

Comment:
Was curious and saw that `wgrib2` has an `-var` option. Might be good to use in case the description could raise some false positive for a variable name to exist in the description.

Reply:
I just tried it a few different ways and couldn't get it to work. It seems like it may be an old option that isn't used anymore?

Reply:
Mine is `v3.0.0 9/2020 ` and works there

Reply:
Also worked with version `2.0.2`

Reply:
We have v0.2.0.8 (which I assume is actually v2.0.8).

Reply:
Have that version on CHPC too and it is present there:
```
$wgrib2 -var ~/scratch/iSnobal/hrrr_CBR/hrrr.20180101/hrrr.t00z.wrfsfcf01.grib2
1:0:HGT
2:142116:TMP
3:231009:RH
4:319902:UGRD
5:399925:VGRD
6:479948:APCP
7:542254:TCDC
8:631147:DSWRF

$ wgrib2 --version
v0.2.0.8 2/2019  Wesley Ebisuzaki, Reinoud Bokhorst, John Howard, Jaakko Hyvätti, Dusan Jovic, Daniel Lee, Kristian Nilssen, Karl Pfeiffer, Pablo Romero, Manfred Schwarb, Gregor Schee, Arlindo da Silva, Niklas Sondell, Sam Trahan, Sergey Varlamov
```

Reply:
Got it, I was using it wrong. The reason we use match is that it allows you to select certain variables, `-var` just lists all of them and doesn't appear to allow you to select. We found the `-match` after talking with someone from NCAR.

Reply:
Correct me here, but the above command does not use any options. Hence my suggestion to add `-var` and limit the output when inspecting for presence of a variable

Reply:
Ah got it, I was thinking that you could use it for extracting variables. Since we already have a variable as `name:height` it was easier to check that the whole variable name was there.